src/index.js

```diff
@@ -24,7 +24,8 @@ export default class ScrollContainer extends PureComponent {
     style: PropTypes.object,
     ignoreElements: PropTypes.string,
     nativeMobileScroll: PropTypes.bool,
-    stopPropagation: PropTypes.bool
+    stopPropagation: PropTypes.bool,
+    ignoredMouseButtons: PropTypes.arrayOf(PropTypes.number)
```

Comment:
I suppose, that it should has name `ignoreMouseButtons` (by analogy with `ignoreElements`).